addons/bestja_frontend_ucw/templates/pages/about_us.xml

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Since Anna will probably want to retain those files somewhere in the repository, I think we should just put them in a separate module.